
%\usepackage[colorlinks=true]{hyperref} % for links and urls
%\usepackage{amsmath,amssymb} % for math
%\usepackage{color,soul} % for highlighting
%\usepackage{booktabs,rotating,multirow} % for nice tables
%\usepackage{authblk} % for author affiliations
%\usepackage{graphicx} % for figures
%\usepackage[numbers]{natbib}
%\usepackage[superscript,biblabel]{cite} % for superscript citations
%\usepackage[above]{placeins} % for FloatBarrier
%\usepackage{amsmath} % for pretty fraction text (heatmap caption)
%\usepackage{lineno} % for line numbers
%\usepackage{geometry} % for supp fig large pages

%% Smaller captions
%\usepackage[font=footnotesize]{caption}

\section*{Abstract}
Hundreds of clinical studies have demonstrated associations between the human microbiome and disease, yet fundamental questions remain on how we can generalize this knowledge.
Results from individual studies can be inconsistent and comparing published data is further complicated by a lack of standard processing and analysis methods.
Here, we introduce the MicrobiomeHD database, which includes 28 published case-control gut microbiome studies spanning ten diseases.
We perform a cross-disease meta-analysis of these studies using standardized methods.
We find consistent patterns characterizing disease-associated microbiome changes.
Some diseases are associated with over 50 genera, while most show only 10 to 15 genus-level changes.
Some diseases are marked by the presence of potentially pathogenic microbes whereas others are characterized by a depletion of health-associated bacteria.
Furthermore, we show that about half of genera associated with individual studies are bacteria which respond to more than one disease.
Thus, many associations found in case-control studies are likely not disease-specific but rather part of a non-specific, shared response to health and disease.

\newpage

\section{Introduction}

The human gastrointestinal tract digests food, absorbs nutrients, and plays important roles in maintaining metabolic homeostasis.
The microbes residing in our gut harvest energy from the food we eat, train our immune system, break down xenobiotics and other foreign products, and release metabolites and hormones important for regulating our physiology \cite{nash-baker, asd-kb, turnbaugh2006obesity}.
Chemical signals from our microbiota can act locally within the gut, and can also have larger systemic effects (e.g. the `gut-brain axis') \cite{Hsiao2013gutbrain,Cryan2012gutbrain,Poutahidis2013gutbrain}.

Due to the physiological interplay between humans and our microbial communities, many diseases are hypothesized to be associated with shifts away from a ``healthy'' gut microbiome.
These include metabolic disorders, inflammatory and auto-immune diseases, neurological conditions, and cancer, among others \cite{nash-baker, turnbaugh2006obesity, asd-son, crc-zhao, par-schep}.
Certain gut-related conditions (e.g. obesity and inflammatory bowel disease) have been extensively studied in human cohorts and in animal experiments, where significant and sometimes causal microbial associations have been shown.
These studies have spurred research into a number of complex diseases with unclear etiologies where a connection to the microbiome is suspected.

Overall, our current understanding of the precise relationships between the human gut microbiome and disease remains limited.
Existing case-control studies often report finding disease-associated microbial ``dysbiosis''.
However, the term ``dysbiosis'' is inconsistently and often vaguely defined, and can have a wide range of interpretations \cite{olesen2016dysbiosis,zaneveld2017karenina}.
Thus, we lack a comprehensive understanding of precisely how microbial communities and specific microbes within those communities cause, respond to, or contribute to disease.
Are different diseases characterized by distinct shifts in the gut microbiome?
Are some diseases marked by an invasion of pathogens whereas others show a depletion of beneficial bacteria?
Can we identify microbial biomarkers for certain conditions, which are consistently enriched or depleted in a disease across many patient cohorts?
Finally, are some bacteria part of a non-specific ``healthy'' or ``diseased'' microbiome and consistently associated with health or disease in general?

One approach to synthesize existing knowledge is to identify consistencies across studies through a meta-analysis, which allows researchers to find and remove false positives and negatives that may obscure underlying biological patterns.
However, prior meta-analyses of case-control gut microbiome studies have yielded mixed results and did not contextualize their findings across multiple diseases \cite{walters2014meta, Sze07092016,finucane2014obesity}.
For some conditions like inflammatory bowel disease (IBD), an overall difference in the gut microbiota was found within several studies, but no individual microbes were consistently associated with IBD across studies \cite{walters2014meta}.
For other conditions like obesity, multiple meta-analyses have found little to no difference in the gut microbiomes of obese and lean patients \cite{walters2014meta,Sze07092016,finucane2014obesity}, even though the microbiome has been causally linked to obesity in mouse models \cite{turnbaugh2006obesity, ridaura2013gut}.
These meta-analyses have been limited by focusing on only one or two diseases, and thus do not extend their findings across a broader landscape of human disease to answer more general questions about overall patterns of disease-associated microbiome shifts.

In this paper, we collected 28 published case-control 16S amplicon sequencing gut microbiome datasets spanning ten different disease states.
We acquired raw data and disease metadata for each study and systematically re-processed and re-analyzed the data.
We investigated whether consistent and specific disease-associated changes in gut microbial communities could be identified across multiple studies of the same disease.
Certain diseases (e.g. colorectal cancer (CRC)) are marked by an enrichment of disease-associated bacteria, while others (e.g. IBD) are characterized by a depletion of health-associated bacteria.
Some conditions (e.g. diarrhea) exhibit large-scale community shifts with many associated microbes, while most show only a handful of associations.
However, many associations are not specific to individual diseases but rather respond to multiple disease states.
In most studies, the majority of the individual disease-associated microbes were part of this set of bacteria that respond non-specifically to healthy and diseased states.
Thus, associations from individual case-control studies should be interpreted with caution, as these microbes may be indicative of a shared response to disease rather than part of disease-specific differences.
Together, these findings reveal distinct categories of dysbioses which can inform the development of microbiome-based diagnostics and therapeutics.

\section{Results}

\subsection{Most disease states show altered microbiomes}

To answer questions about the reproducibility and generalizability of reported associations between the human microbiome and disease, we collected, re-processed, and re-analyzed raw data from a collection of microbiome datasets.
We included studies with publicly available 16S amplicon sequencing data (i.e. FASTQ or FASTA) for stool samples from at least 15 case patients which also had associated disease metadata (i.e. case or control disease labels).
Studies which exclusively focused on children under 5 years old were excluded from our analyses.
We identified over 50 suitable case-control 16S datasets, of which 28 were successfully downloaded, processed, and included in a publicly available database which we called MicrobiomeHD\cite{microbiomehd}.
Characteristics of these datasets, including sample sizes, diseases and conditions, and references, are shown in Table \ref{tab:datasets} and Supplementary Table \ref{tab:datasets_full_info}.
For each downloaded study, we processed the raw sequencing data through our 16S processing pipeline (\url{https://github.com/thomasgurry/amplicon_sequencing_pipeline}) (see Supplementary Tables \ref{tab:processing} and \ref{tab:data} for detailed data sources and processing methods).
100\% denovo OTUs were assigned taxonomy with the RDP classifier \cite{wang2007naive} (c $=$ 0.5), converted to relative abundances by dividing by total sample reads, and collapsed to the genus level. OTUs which were not assigned at the genus level were discarded.
By collapsing data to the genus level, we lost the sensitivity to detect fine-scale differences in species or strain abundances across case and control groups, but we minimized certain batch effects that plague comparisons across studies.
Thus, we took a course-grained approach to optimize our ability to compare data across studies at the expense of phylogenetic resolution.

We first asked whether reported associations between the gut microbiome and disease would be recapitulated once we controlled for processing and analysis approaches.
To test whether the gut microbiome is altered in a variety of disease states, we built genus-level random forest classifiers to classify cases from controls within each study.
We compared the resulting area under the Receiver Operating Characteristic (ROC) curves (AUC) across studies (Figure \ref{fig:fig1}A, Supplementary Figure \ref{fig:roc_curves}).
We could classify cases from controls (AUC $>$ 0.7) for at least one dataset for all diseases except arthritis and Parkinson's disease, which each only had one study.
Notably, all diarrhea datasets (except Youngster et al. (2014) \cite{cdi-youngster}, which had only 4 distinct control patients and thus was not included in this analysis) had very high classifiability (AUC $>$ 0.9).
We successfully classified patients from controls (AUC $>$ 0.7) in three out of four IBD studies and all four CRC studies, which is consistent with previous work showing that these patients can be readily distinguished from controls using supervised classification methods \cite{walters2014meta,ibd-papa,crc-baxter,crc-zeller}.
Thus, the microbiome is indeed altered in many different diseases.

\subsection{Loss of beneficial microbes or enrichment of pathogens}

We next wondered whether the specific type of alteration was consistent across independent cohorts of patients with the same disease.
We performed univariate tests on genus-level relative abundances for each dataset independently and compared results across studies (Kruskal-Wallis (KW) test with the Benjamini-Hochberg false discovery rate (FDR) correction \cite{benjamini-hochberg}).
Our re-analyses of the studies were largely consistent with the originally reported results.
The same taxonomic groups showed similar trends as in the original publications, despite differences in data-processing methodologies (see Supplementary Note \ref{sec:lit_comp} for a full comparison of our re-analysis with previously published results).
Furthermore, we found that the disease-associated changes in the microbiome could be categorized into meaningful groups which provide insight into possible etiologies or therapeutic strategies for different types of disease.

In some diseases, microbiome shifts are dominated by an enrichment of a small number of ``pathogenic'' bacteria.
In these cases, it is possible that the microbes play a causal role and that they could be targeted with narrow-spectrum antimicrobials.
Colorectal cancer is characterized by such a shift, and we found significant agreement across three of the four CRC studies \cite{crc-zhao,crc-baxter,crc-zeller,crc-xiang} (Figures \ref{fig:fig1}B and \ref{fig:fig2}, genus labels in Supplementary Figure \ref{fig:supp_dis_specific}).
Dysbiosis associated with CRC is generally characterized by increased prevalence of the known pathogenic or pathogen-associated \textit{Fusobacterium}, \textit{Porphyromonas}, \textit{Peptostreptococcus}, \textit{Parvimonas}, and \textit{Enterobacter} genera (i.e. these genera were higher in CRC patients in 2 or more studies, Figures \ref{fig:fig2} and \ref{fig:fig3}A, genus labels in Supplementary Figures \ref{fig:supp_dis_specific} and \ref{fig:supp_meta_heatmap}).
\textit{Fusobacterium} is associated with a broad spectrum of human diseases and \textit{Porphyromonas} is a known oral pathogen \cite{Han2015fusopatho, Flynn2016oral}.

By contrast, other disease-associated microbiome shifts are characterized by a depletion of health-associated bacteria in patients relative to controls.
In these cases, probiotics that replace missing taxa may be a better treatment strategy than anti-microbials.
Across our four IBD studies, patient microbiomes were dominated by a depletion of genera in patients relative to controls, especially butyrate-producing \textit{Clostridiales} \cite{ibd-papa,ibd-gevers,ibd-hut,ibd-engstrand} (Figures \ref{fig:fig1}B and \ref{fig:fig2}, genus labels in Supplementary Figure \ref{fig:supp_dis_specific}).
In particular, five genera from the \textit{Ruminococcacaea} and \textit{Lachnospiracaea} families were consistently depleted in IBD patients relative to controls in at least two studies (Figure \ref{fig:fig3}A, genus labels in Supplementary Figure \ref{fig:supp_meta_heatmap}).
While not all genera within \textit{Ruminococcacaea} and \textit{Lachnospiracaea} are verified short-chain fatty acid (SCFA) producers, the dominant genera within these families are known to harbor genes for short chain fatty acid production \cite{louis2010diversity} and are often associated with colonic health \cite{flint2012role,miquel2013faecalibacterium,reeves2012suppression}.
We found similar results when comparing Crohn's disease and ulcerative colitis patients to controls separately, without any consistent patterns across datasets that distinguished either IBD sub-type (Supplementary Note \ref{sec:split_cases}; Supplementary Figures \ref{fig:split_cases_fig1} and \ref{fig:split_cases_heatmaps}).

Some conditions are characterized by a broad restructuring of gut microbial communities.
In these cases, full community restoration strategies like fecal microbiota transplants may be more appropriate.
For example, diarrhea consistently results in large-scale rearrangements in the composition of the gut microbiome, which is likely reflective of reduced stool transit time (Figures \ref{fig:fig1} and \ref{fig:fig2}).
We saw many microbes consistently associated with both \textit{Clostridium difficile} infection (CDI) and non-CDI diarrhea (Figures \ref{fig:fig2} and \ref{fig:fig3}A) \cite{cdi-youngster,cdi-schubert,cdi-vincent,edd-singh}.
In general, Proteobacteria increase in prevalence in patients with diarrhea, with a concomitant decrease in the relative abundances of Bacteroidetes and some Firmicutes.
In particular, we see a reduction in butyrate-producing Clostridia, including genera within \textit{Ruminococcaceae} and \textit{Lachnospiraceae} families, which have been associated with a healthy gut \cite{wong2006colonic}.
We also see an increase in prevalence of genera that contain organisms often associated with lower pH and higher oxygen levels of the upper-gut, like \textit{Lactobacillaceae} and \textit{Enterobacteriaceae}, in patients with diarrhea (Figure \ref{fig:fig3}A) \cite{donaldson2016gut}.
Additionally, both CDI and non-CDI diarrhea patients had lower alpha diversity, a measure of overall community structure, than healthy controls in all studies (Supplementary Figures \ref{fig:fig_alpha}-\ref{fig:fig_alpha_simpson}).
Consistent with the CDI and non-CDI diarrheal studies, we also found that organisms associated with the upper gut, like \textit{Lactobacillus} and \textit{Enterobacteriaceae}, appear to be enriched in IBD patients, who can present with diarrheal symptoms (Supplementary Figure \ref{fig:supp_dis_specific}) \cite{donaldson2016gut,Kirsner1982ibd}.
IBD patients also tended to have lower alpha diversities than controls (Crohn's disease vs. controls in three studies, ulcerative colitis vs. controls in two studies; Supplementary Figures \ref{fig:fig_alpha}-\ref{fig:fig_alpha_simpson}), though this difference was less drastic than in the diarrheal studies where all patients had active diarrhea.

In some studies, confounding variables may drive associations.
For example, there were no consistent differences between cases and controls across HIV studies because of demonstrated confounders \cite{noguera2016gut,lozupone2013alterations,hiv-dinh} (Figure \ref{fig:fig2}, \ref{fig:fig3}A).
As in the original Lozupone et al. (2013) \cite{lozupone2013alterations} study, we found enrichment in \textit{Prevotella}, \textit{Catenibacterium}, \textit{Dialister}, and \textit{Desulfovibrio} in HIV-positive patients, in addition to 8 other genera (Figure \ref{fig:fig2} and Supplementary Figure \ref{fig:supp_dis_specific}).
We also found depletion of \textit{Bacteroides}, \textit{Odoribacter}, \textit{Anaerostipes}, \textit{Parasutterella}, and \textit{Alistipes} in HIV-positive patients relative to controls.
However, the Noguera-Julian et al. (2016) study showed that the genera that were significantly associated with HIV in the Lozupone paper were strongly associated with sexual behavior (e.g. men who have sex with men were associated with much higher \textit{Prevotella} levels), and our re-analysis also found conflicting results between these two studies (Figure \ref{fig:fig2}).
Thus, there is no consensus on what genera are associated with HIV.
Obesity is another example where confounding variables may drive microbiome alterations.
Three recent meta-analyses found no reproducible obesity-associated microbiome shifts \cite{walters2014meta,Sze07092016,finucane2014obesity}, which is consistent with our classification results where we were only able to accurately classify obese and control patients in two out of five studies (Zhu et al. (2013) \cite{nash-baker}, Turnbaugh et al. (2009) \cite{ob-gordon}; Figure \ref{fig:fig1}A).
Our genus-level re-analysis did find a few consistent genus-level associations between lean and obese patients \cite{nash-baker,ob-gordon,ob-goodrich,ob-zupancic,ob-ross}.
Two genera, \textit{Roseburia} and \textit{Mogibacterium}, were significantly enriched in obese individuals across two of the obesity studies (Figure \ref{fig:fig3}A).
Furthermore, \textit{Anaerovorax}, \textit{Oscillibacter}, \textit{Pseudoflavonifractor}, and \textit{Clostridium IV}  were depleted in obese patients relative to controls in two of the studies.
However, two of the five studies had no significant genus-level associations (q $<$ 0.05), despite one having a large sample size (Zupancic et al. (2012) \cite{ob-zupancic}).
This suggests that confounding factors like diet may have given rise to certain associations found in our re-analysis and previously reported in the literature \cite{finucane2014obesity}.
More studies that control for potential confounders, like host behavior and diet, will be required for diseases like obesity and HIV, where associations with the microbiome remain unclear.
Finally, patients in case-control cohorts are frequently on other medications such as antibiotics which may confound disease-associated microbiome shifts.
Six of our datasets included antibiotics metadata, and of these only one dataset (Schubert et al, 2014 \cite{cdi-schubert}) had more than 5 controls who were on antibiotics.
Thus, it is very likely that disease-associated genera in conditions which are often treated with antibiotics (e.g. diarrhea, IBD) are confounded with antibiotic usage.
Future case control studies should focus on better separating treatment and disease variables by collecting detailed metadata on antibiotic and other medication usage, and perhaps also by recruiting controls undergoing a variety of treatments.

\subsection{Shared vs. disease-specific microbial responses}

Finally, we sought to determine whether a unified microbiome response to general health and disease could be identified.
Previous studies have proposed that reduced alpha diversity is a reliable indicator of disease-associated dysbiosis \cite{ob-gordon,cdi-vincent,mosca2016gut}.
In our re-analysis, we found no consistent reduction of alpha diversity in case patients, with the exception of diarrhea and perhaps IBD (Supplementary Figures \ref{fig:fig_alpha}-\ref{fig:fig_alpha_simpson}).
These results are consistent with previous meta-analyses, which found inconsistent relationships between alpha diversity and disease and very small effect sizes in non-diarrheal diseases \cite{walters2014meta, Sze07092016}.
To further address the question of whether we could find a robust, generalized signal for diseased microbiomes regardless of the disease type, we built random forest classifiers to distinguish healthy patients from any type of case patient.
The AUCs from these general healthy vs. disease classifiers correlated strongly with the original single-dataset classification results, indicating that there is indeed a general microbiome signal that can be identified even across different diseases (see Supplementary Note \ref{sec:overall_classifier} and Supplementary Figure \ref{fig:overall_classifier}).

Having putatively shown the presence of a generalized microbial response to disease, we next sought to identify individual genera which respond non-specifically to health and disease.
We considered a genus to be part of the non-specific, shared microbial response if it was significantly enriched or depleted (q $<=$ 0.05) in at least one dataset from at least two different diseases (see Supplementary Note \ref{sec:core_defns} and Supplementary Figures \ref{fig:core_defns} and \ref{fig:core_sig} for further discussion on alternative definitions and statistical significance of shared response).
We identified 24 health-associated genera and 20 disease-associated genera out of the 152 genera that were significant in at least one dataset (Figure \ref{fig:fig3}A, genus labels in Supplementary Figure \ref{fig:supp_meta_heatmap}).
We also found 7 genera that were both health- and disease-associated (i.e. they were enriched in controls across at least two diseases, but were also depleted in controls in different comparisons across at least two diseases) (Figure \ref{fig:fig3}A, black).
Perhaps these genera represent bacteria disproportionately affected by confounders or technical artifacts.
Alternatively, different species or strains within these genera may play alternate roles across diseases or community contexts, giving rise to variable responses at the genus level.

We identified distinct sub-groups of microbes within the \textit{Bacteroidetes} and \textit{Firmicutes} phyla which respond non-specifically to health and disease (Figure \ref{fig:fig3}A).
The order \textit{Clostridiales} (specifically the \textit{Lachnospiraceae} and \textit{Ruminococcacaea} families) is associated with health across multiple diseases while the order \textit{Lactobacillales} and family \textit{Clostridiales Incertae Sedis XI} are associated with disease.
The majority of the the non-specific responders in the order \textit{Clostridiales} were associated with health, comprising the majority of all of the microbes which were non-specifically associated with healthy patients (17 genera out of 24 total health-associated genera).
All of five of the non-specific responders in the order \textit{Lactobacillales} were enriched in case patients across multiple diseases.
\textit{Lactobacillales} genera are adapted to the lower pH of the upper gastrointestinal tract \cite{donaldson2016gut}.
Perhaps the shared disease-associated taxa are indicators of shorter stool transit times and disruptions in the redox state and/or pH of the lower intestine, rather than specific pathogens.
These non-specific responders are consistent with the results from a recent meta-analysis of six metagenomics datasets, which also found \textit{Lactobacillales} and \textit{Clostridiales} microbes among the most discriminative classification features across multiple studies \cite{pasolli2016machine}.
Finally, we found that the order \textit{Bacteroidales} is more mixed: two \textit{Bacteroidales} genera were non-specifically associated with health, one with disease, and two with both health and disease.

A majority of bacterial associations within individual studies overlap with the shared response.
For each dataset that had at least one significant (q $<$ 0.05) association, we calculated the percent of associated genera which were also part of the non-specific response in the same direction (Figure \ref{fig:fig3}B).
Strikingly, the majority of microbial responses were not specific to individual diseases; on average, 51\% of a dataset's genus-level associations were genera which were associated with more than one disease.
In light of this finding, it is important that researchers performing future case-control studies consider whether an identified microbial association is truly specific to their disease of interest or is instead responding to a common symptom (e.g. diarrhea) or perhaps generally associated with health or sickness.
Additionally, they can use the knowledge that many microbes respond non-specifically to disease to narrow lists of putative causal or diagnostic biomarkers to microbes which fall outside of the shared response and are thus more likely to be specific to the disease being studied.
Researchers can access an updated list of shared microbial responders from this analysis at the MicrobiomeHD database \cite{microbiomehd}, or they can curate their own lists by performing similar cross-disease meta-analyses.

Bacteria which are non-specifically associated with health are both ubiquitous and abundant across people, whereas bacteria which are non-specifically associated with disease are abundant when present but are not ubiquitous.
We calculated the average relative abundance (i.e. the total relative abundance across all patients divided by the number of patients with non-zero abundance) and ubiquity (i.e. the number of patients with non-zero abundance divided by the total number of patients) for each genus in the shared response.
We found that health-associated genera were more ubiquitous than disease-associated ones, but not necessarily more abundant (Figure \ref{fig:fig3}C).
Thus, presence/absence of the non-specifically disease-associated genera appears to be a better indicator of disease-associated microbial shifts than changes in their relative abundances.
However, a small subset of the non-specifically disease-associated genera were relatively ubiquitous across patients.
Among the most ubiquitous were \textit{Escherichia/Shigella} and \textit{Streptococcus}.
\textit{Escherichia} includes common commensal strains, as well as pathogenic strains \cite{rasko2008pangenome}, and is frequently present in healthy people's guts as well as over-represented in sick patients. Genera within \textit{Enterobacteriaceae}, \textit{Lactobacillaceae}, and \textit{Streptococcaceae} families are dominant in the upper gastrointestinal tract \cite{donaldson2016gut,wang2013upper} and are present in many people's stool at low frequency. These taxa likely become enriched with faster stool transit time (i.e. signatures of diarrhea) \cite{donaldson2016gut,savage1977microbial}.


\subsection{Within and cross-disease meta-analysis improves interpretability}

Identifying disease-specific and non-specific microbial responses required comparing studies both within and across multiple diseases.
Multiple studies of the same disease were necessary to identify shifts consistently associated with individual diseases.
We did not find consistent bacterial associations for conditions with fewer than four datasets (Figure \ref{fig:fig1}, \ref{fig:fig3}A).
Within-disease meta-analysis also increased our ability to interpret the results from any one dataset.
Despite few significant differences, some of these studies (e.g. Zhang et al. (2013) \cite{mhe-zhang}, Zhu et al. (2013) \cite{nash-baker}) had high classifiability of patients vs. controls (AUC $>$ 0.7, Figure \ref{fig:fig1}A), indicating that there may be a disease-associated shift that was not detected by univariate comparisons.
However, because few other studies of the same disease were available for comparison, we could not confidently interpret the classification results beyond the reported AUC.
For other studies with high AUCs but few univariate associations (e.g. Vincent et al. (2013) \cite{cdi-vincent}, Morgan et al. (2012) \cite{ibd-hut}, Chen et al. (2012) \cite{crc-xiang}), our confidence that the high AUCs reflect true disease-associated differences increased because the high AUCs were consistent with other classifiers from the same disease type.

Meta-analysis identified potential false positives and false negatives across studies and conditions.
For example, we found that reported associations between alpha diversity and disease within individual studies tended to lose significance when looking across studies, except in the case of diarrhea and perhaps IBD (Supplementary Figures \ref{fig:fig_alpha}-\ref{fig:fig_alpha_simpson}).
Another example of a potential false positive was the association between \textit{Prevotella} and disease.
Autism \cite{asd-kb}, rheumatoid arthritis \cite{ra-littman}, and HIV \cite{lozupone2013alterations,hiv-dinh} have each been reported to be associated with \textit{Prevotella}. For each of these diseases, the associations with \textit{Prevotella} were weakly significant or complicated by confounding factors.
In our more statistically conservative re-analysis, we found no association between autism or arthritis and \textit{Prevotella}.
As mentioned previously, in the case of HIV, the association with \textit{Prevotella} was due to demographic factors unrelated to disease \cite{noguera2016gut}.
Regardless of whether shifts in \textit{Prevotella} are truly biologically related to each studied disease state, it is clear that such shifts are not specific to one particular condition and should not be reported as putative disease-specific biomarkers.
We also found that certain signals picked out by meta-analysis did not always hold within individual studies.
For example, studies with small sample sizes often had few or no significant associations (e.g. Vincent et al. (2014) \cite{cdi-vincent}, Chen et al. (2012) \cite{crc-xiang}, and Willing et al. (2009) \cite{ibd-engstrand}).
Here, the fact that other studies analyzing the same diseases consistently found associations strengthens the hypothesis that the lack of microbiome-associated signal in these studies was due to low power rather than a lack of true signal.
Because individual studies are plagued by low statistical power, confounding variables, and batch effects which can obscure biological signals, the identification of disease-specific and non-specific microbial associations will continue to improve as more datasets and diseases are included in future meta-analyses.


\section{Discussion}

Here, we report patterns of disease-associated shifts in the human gut microbiome which differ in their directionality (i.e. fraction of disease-enriched vs. disease-depleted genera) and extent (i.e. total number of genera that differ between cases and controls).
Some diseases are characterized by an invasion of pathogenic or disease-associated bacteria (e.g. CRC), while others largely show a depletion of health-associated microbes (e.g. IBD).
Diarrheal illnesses induce large-scale rearrangement of many members of the microbiota, whereas other conditions show fewer associations.
We also find a set of microbes which are non-specifically associated with multiple diseases and show that these microbes comprise many of the disease-associated genera within any given study.

The identification of a non-specific microbial response is an important concept that should be considered in future case-control microbiome studies.
It suggests that studies should be interpreted with extra caution, as many identified microbial associations may be indicative of a shared response to health or disease rather than a disease-specific biological difference.
Microbes that are non-specifically associated with multiple diseases would not be useful as disease-specific diagnostics or to address causality \cite{olesen2016dysbiosis}.
On the other hand, bacteria that are associated with healthy patients across multiple diseases could be developed into a general probiotic which may be suited for many different conditions.

Additionally, characterizing ``dysbioses'' by their directionality and extent is a useful framework to generate hypotheses for future research on complex, heterogenous diseases with links to the microbiome.
For example, the search for microbiome-based diagnostics may be more appropriate for diseases with consistently enriched disease-associated microbes, like CRC.
On the other hand, patients with diseases which are characterized by depletion of health-associated microbes, like IBD, may benefit from prebiotic or probiotic interventions designed to enrich for these taxa.
Furthermore, conditions which are characterized by large-scale shifts in community structure may be well-suited to treatment with fecal microbiota transplantation, as in CDI \cite{cdi-youngster}.
While many of these conditions are unlikely to be fully treated by antibiotics, probiotics, or fecal microbiota transplants, our proposed framework could guide the search for new therapies and etiologies by generating testable hypotheses with higher likelihoods of success \cite{olesen2016dysbiosis}.

This analysis is the first to compare microbiome studies across more than two different diseases and highlights the importance of making raw data and associated patient metadata publicly available to enable future, more comprehensive analyses.
This analysis does not include all possible studies, and certain important gastrointestinal diseases (e.g. irritable bowel syndrome) are missing, largely due to data and metadata availability.
Future studies should expand on this work by including more cohorts from the same diseases as well as more diseases.
To re-analyze these studies, we applied standard methods commonly used in the field and assumed that the original study designs and patient selection methods were adequate.
We were reassured to find that a straightforward and standardized approach was able to recover very similar results to those previously reported in the various papers.
Thus, we did not formally investigate heterogeneity between cohorts or technical inter-study batch effects.
However, it is clear from our genus-level results that there is significant variation even across studies of the same disease.
There are many possible reasons for this variation (experimental and sequencing artifacts, host-related covariates, stochastic disease-associated community changes etc. \cite{ zaneveld2017karenina,Falony2016variation, David2014lifestyle}), and future analyses should consider methods to correct for host confounders and technical batch effects.
Concerns about batch effects motivated us to analyze the data at the genus level, which necessarily limited our resolution and biological interpretations of identified associations (e.g. different species or strains within a genus may have different associations with disease, which would not be captured in this analysis).
Making raw data from case-control studies publicly available will also allow researchers to develop methods to correct for these batch effects, in addition to enabling more comprehensive future meta-analyses.

Despite the limitations of this study, our results provide more nuanced insight into dysbiosis, revealing distinct types of alterations that more precisely describe disease-associated microbiome shifts.
As the number of case-control cohorts increases, similar meta-analyses could be used to compare related diseases and identify microbiome alterations associated with general host physiological changes.
For example, there may be a group of microbes which respond to or cause systemic inflammation.
Could we identify these microbes by comparing multiple inflammatory or auto-immune diseases and study them to better understand the interactions between the microbiome and our immune system?
Furthermore, some microbes may be consistently associated with neurological conditions and could contribute to the gastrointestinal symptoms that accompany or precede neurological manifestations \cite{asd-kb,par-schep}.
Studying these microbes could help us understand the `gut-brain axis' by identifying common neuroactive molecules produced by these bacteria, which could also be used as targets for new treatments \cite{Hsiao2013gutbrain,Cryan2012gutbrain,Poutahidis2013gutbrain}.
Finally, meta-analysis could be used to identify subsets of patients who exhibit distinct microbiome shifts within heterogenous diseases like IBD or in conditions which exhibit stochastic microbial responses, allowing for further stratification of disease subtypes and microbiome disruptions \cite{zaneveld2017karenina,ibd-engstrand,Pascal2017crohns}.
This work demonstrates that employing standard methods to contextualize new results within the broader landscape of clinically relevant microbiome studies is feasible and adds value to individual analyses.
As excitement in this field grows, researchers should harness the increasing number of replicated case-control studies to swiftly and productively advance microbiome science from putative associations to transformative clinical impact.


\section{Methods}

\subsection{Dataset collection}

We identified case-control 16S studies from keyword searches in PubMed and by following references in meta-analyses and related case-control studies.
We included studies with publicly available raw 16S data (fastq or fasta) and metadata indicating case or control status for each sample.
Most data was downloaded from online repositories (e.g. SRA) or links provided in the original publications, but some were acquired after personal communication with the authors (Supplementary Table \ref{tab:data}).
We did not include any studies which required additional ethics committee approvals or authorizations for access (e.g. controlled dbGaP studies).
In studies where multiple body sites were sampled or where multiple samples were taken per patient, we also required the respective metadata to include those metadata.
We analyzed only stool 16S samples, and excluded studies with fewer than 15 case patients.
In CRC studies with multiple control groups (e.g. healthy and non-CRC adenoma), only the healthy patients were used as controls for all of our comparisons.
In studies with non-healthy controls (e.g. non-IBD patients), these patients were used as controls (as in the original papers).
In the Schubert et al. CDI study \cite{cdi-schubert}, which had both CDI and non-CDI diarrheal patients, each group was used as an independent case group compared with controls.
We also analyzed the NASH and obese patients from the Zhu et al. study \cite{nash-baker} as independent case groups.
When obesity studies reported body mass index instead of obesity status, we considered patients with BMI less than 25 as our control group and patients with BMI greater than 30 as the case group.

\subsection*{16S processing}
Raw data were downloaded and processed through our in-house 16S processing pipeline ({\url{ https://github.com/thomasgurry/amplicon_sequencing_pipeline}).
Data and metadata were acquired as described in Supplementary Table \ref{tab:data}.
When needed, we de-multiplexed sequences by finding exact matches to the provided barcodes and trimmed primers with a maximum of 1 mismatch.
In general, sequences were quality filtered by truncating at the first base with Q $<$ 25.
However, some datasets did not pass this stringent quality threshold (i.e. the resulting OTU table was either missing many of the original samples, or the read depth was significantly lower than reported in the original paper).
For 454 data, we loosened the quality threshold to 20, whereas for paired-end Illumina data we removed reads with more than 2 expected errors.
If possible, all reads were trimmed to 200 bp.
In cases where this length trimming discarded a majority of sequences, we lowered our threshold to 150 or 101 bp.
The specific processing parameters we used for each dataset can be found in Supplementary Table \ref{tab:processing}.
To assign OTUs, we clustered OTUs at 100\% similarity using USEARCH \cite{edgar-usearch-2010} and assigned taxonomy to the resulting OTUs with the RDP classifier \cite{wang2007naive} and a confidence cutoff of 0.5.
For each dataset, we removed samples with fewer than 100 reads and OTUs with fewer than 10 reads, as well as OTUs which were present in fewer than 1\% of samples within a study.
We calculated the relative abundance of each OTU by dividing its value by the total reads per sample.
We then collapsed OTUs to genus level by summing their respective relative abundances, discarding any OTUs which were unannotated at the genus level.
All statistical analyses were performed on this genus-level relative abundance data.

\subsection{Statistical analyses}

To perform supervised classification of cases and controls within each dataset, we built Random Forest classifiers with 5-fold cross-validation.
To build our train and test sets, we used the python \texttt{scikit-learn} \texttt{StratifiedKFold} function with shuffling of the data \cite{scikit-learn}.
To build our classifiers, we used the \texttt{RandomForestClassifier} function with 1000 estimators and other default settings \cite{scikit-learn}.
We found no significant effect of various Random Forest parameters on the AUCs  (Supplementary Figures \ref{fig:rf_params_gini} and \ref{fig:rf_params_entropy}).
We calculated the interpolated area under the ROC curve (AUC) for each classifier based on the cross-validation testing results.
To account for spurious high classifiability due to class imbalances, we also calculated the Cohen's kappa score for each classifier using \texttt{sklearn.metrics.cohen\_kappa\_score} on the test set predictions (Supplementary Table \ref{tab:kappa}).
The kappa scores correlated well with the AUCs (Pearson $\rho$ $=$ 0.9), indicating that the majority of the classifiers performed well even when considering their underlying data distributions.
We excluded Youngster et al. (2014) \cite{cdi-youngster}, which had only 4 distinct control patients, from all classifier analyses.

We performed univariate analyses on the relative abundances of genera in cases and controls with a non-parametric Kruskal-Wallis test using the \\ \texttt{scipy.stats.mstats.kruskalwallis} function \cite{scipy}.
We corrected for multiple hypothesis testing in each dataset with the Benjamini-Hochberg false discovery rate \cite{benjamini-hochberg}.
We performed all analyses on genus-level relative abundances for each dataset individually, and then compared these results across all studies.

We considered a genus to be consistently associated with a disease (Figure \ref{fig:fig3}A, bottom) if it was significantly associated (q $<$ 0.05) with the disease in the same direction in at least two studies of that disease.
We considered a genus to be a non-specific microbial association (Figure \ref{fig:fig3}A, top) if it was significantly associated (q $<$ 0.05) in at least one dataset of at least two different diseases in the same direction.
When we defined these non-specific genera, we did not include datasets which used non-healthy controls (Papa et al. (2012) \cite{ibd-papa} and Gevers et al. (2014) \cite{ibd-gevers}) and the Lozupone et. al (2013) dataset \cite{lozupone2013alterations}, where the microbiome signal reflected behavior rather than disease state \cite{noguera2016gut}.

To build our generalized healthy vs. disease classifiers, we first concatenated metadata and genus-level abundance data for all datasets which had healthy controls (i.e. all datasets except Papa et al. (2012) \cite{ibd-papa} and Gevers et al. (2014) \cite{ibd-gevers}, which used non-IBD patients as controls, and CDI Youngster (2014), \cite{cdi-youngster}, which had only 4 distinct controls).
We performed leave-one-dataset-out and leave-one-disease-out cross validation and calculated an AUC for each of the testing results.

\subsection{Microbiome community analyses}

Alpha diversities were calculated based on the non-collapsed 100\% OTU-level relative abundances, and included un-annotated OTUs.
We calculated alpha diversity metrics with the \texttt{skbio.math.diversity.alpha.chao1}, \texttt{shannon}, and \texttt{simpson} implementations.

We calculated the average abundance and ubiquity (Figure \ref{fig:fig3}C) of each genus as the mean of its average values in each dataset across all patients.
To calculate the abundance of each genus, we first calculated each genus's mean abundance within each dataset.
We counted only patients with non-zero abundance of the genus in this calculation.
We then took the average of these mean abundances across all datatsets.
To calculate the ubiquity of each genus, we calculated the percent of patients with non-zero abundance of that genus in each dataset.
We then took the average of these ubiquities across all datasets.

\subsection{Code availability}

The code to reproduce all of the analyses in this paper is available at \url{https://github.com/cduvallet/microbiomeHD}.
We encourage researchers to incorporate their existing and future case-control studies into the MicrobiomeHD database by contacting us.

\subsection{Data availability}

Raw sequencing data for each study can be accessed as described in Supplementary Table \ref{tab:data}.
The raw processed OTU tables can be accessed at the MicrobiomeHD database, available at \url{https://doi.org/10.5281/zenodo.840333} \cite{microbiomehd}.

Supplementary files, including the q-values for all genus-level comparisons in every dataset, disease-associated genera for the diseases with more than three datasets, and a list of non-specific genera are also available at \url{https://github.com/cduvallet/microbiomeHD}.
All other relevant data supporting the findings of the study are available in this article and its Supplementary Information files, or from the corresponding author upon request.

\section{Acknowledgements}

We thank the authors who made their data publicly available as well as those who provided data after personal communication.
The Center for Microbiome Informatics and Therapeutics supported the computational resources for this project, and C.D. acknowledges support by the Department of Defense through the National Defense Science \& Engineering Graduate Fellowship (NDSEG) Program.
We thank Shijie Zhao and Xiaoqian Yu for their early contributions to the literature review, as well as Nathaniel Chu and other members of the Alm and Irizarry groups for helpful discussions.

\section{Author contributions}

C.D., T.G., and E.A. conceived of the research.
C.D., S.G., and T.G. identified and downloaded the datasets.
C.D. and T.G. wrote the code to process data.
C.D. processed the data and performed all analyses.
C.D., S.G, R.I., and E.A. interpreted the results and prepared the manuscript.

\subsection{Competing Financial Interests}

E.A. is on the Board of Directors of OpenBiome, a non-profit stool bank. E.A. is a co-founder of Finch Therapeutics, which aims to develop microbiome-based therapeutics.
The remaining authors declare no competing financial interests.

\FloatBarrier

\section{Tables}

{
\renewcommand{\arraystretch}{1.1}
\begin{table}[h]
%\resizebox{\textwidth}{!}{\begin{tabular}{|c|c|c|c|c|c|c|c|c|c|}
\resizebox{\textwidth}{!}{\begin{tabular}{c c c c c c}
	\hline
	\textbf{Dataset ID} & \textbf{Controls} & \textbf{N (controls)} & \textbf{Cases} & \textbf{N (cases)} & \textbf{Ref.} \\
	\hline
	Singh 2015, EDD & H & 82 & EDD & 201 & \cite{edd-singh} \\
	Schubert 2014, CDI & H & 154 & CDI & 93 & \cite{cdi-schubert} \\
	Schubert 2014, nonCDI & H & 154 & nonCDI & 89 & \cite{cdi-schubert} \\
	Vincent 2013, CDI & H & 25 & CDI & 25 & \cite{cdi-vincent} \\
	Youngster 2014, CDI & H & 4 & CDI & 19 & \cite{cdi-youngster} \\
	Goodrich 2014, OB & H & 428 & OB & 185 & \cite{ob-goodrich} \\
	Turnbaugh 2009, OB & H & 61 & OB & 195 & \cite{ob-gordon} \\
	Zupancic 2012, OB & H & 96 & OB & 101 & \cite{ob-zupancic} \\
	Ross 2015, OB & H & 26 & OB & 37 & \cite{ob-ross} \\
	Zhu 2013, OB & H & 16 & OB & 25 & \cite{nash-baker} \\
	Baxter 2016, CRC & H & 172 & CRC & 120 & \cite{crc-baxter} \\
	Zeller 2014, CRC & H & 75 & CRC & 41 & \cite{crc-zeller} \\
	Wang 2012, CRC & H & 54 & CRC & 44 & \cite{crc-zhao} \\
	Chen 2012, CRC & H & 22 & CRC & 21 & \cite{crc-xiang} \\
	Gevers 2014, IBD & nonIBD & 16 & CD & 146 & \cite{ibd-gevers} \\
	Morgan 2012, IBD & H & 18 & UC, CD & 108 & \cite{ibd-hut} \\
	Papa 2012, IBD & nonIBD & 24 & UC, CD & 66 & \cite{ibd-papa} \\
	Willing 2010, IBD & H & 35 & UC, CD & 45 & \cite{ibd-engstrand} \\
	Noguera-Julian 2016, HIV & H & 34 & HIV & 205 & \cite{noguera2016gut} \\
	Dinh 2015, HIV & H & 15 & HIV & 21 & \cite{hiv-dinh} \\
	Lozupone 2013, HIV & H & 13 & HIV & 23 & \cite{lozupone2013alterations} \\
	Son 2015, ASD & H & 44 & ASD & 59 & \cite{asd-son} \\
	Kang 2013, ASD & H & 20 & ASD & 19 & \cite{asd-kb} \\
	Alkanani 2015, T1D & H & 55 & T1D & 57 & \cite{t1d-alkanani} \\
	Mejia-Leon 2014, T1D & H & 8 & T1D & 21 & \cite{t1d-mejia} \\
	Wong 2013, NASH & H & 22 & NASH & 16 & \cite{nash-chan} \\
	Zhu 2013, NASH & H & 16 & NASH & 22 & \cite{nash-baker} \\
	Scher 2013, ART & H & 28 & PSA, RA & 86 & \cite{ra-littman} \\
	Zhang 2013, LIV & H & 25 & CIRR, MHE & 46 & \cite{mhe-zhang} \\
	Scheperjans 2015, PAR & H & 74 & PAR & 74 & \cite{par-schep} \\
	\hline
\end{tabular}}
\caption{Datasets collected and processed through standardized pipeline. Disease labels: ART = arthritis, ASD = austism spectrum disorder, CD = Crohn's disease, CDI = \textit{Clostridium difficile} infection, CIRR = liver cirrhosis, CRC = colorectal cancer, EDD = enteric diarrheal disease, H = healthy, HIV = human immunodeficiency virus, LIV = liver diseases,  MHE =  minimal hepatic encephalopathy, NASH = non-alcoholic steatohepatitis, OB = obesity, PAR = Parkinson's disease, PSA = psoriatic arthritis, RA = rheumatoid arthritis, T1D = type I diabetes, UC = ulcerative colitis. nonCDI controls are patients with diarrhea who tested negative for \textit{C. difficile} infection. nonIBD controls are patients  with gastrointestinal symptoms but no intestinal inflammation. Datasets are ordered as in Figure \ref{fig:fig1}.}\label{tab:datasets}
\end{table}
}

\section{Figures}

\newpage
\begin{figure}[h]
	\begin{center}
  	\makebox[\textwidth][c]{\includegraphics[width=1\textwidth]{samplesize_auc_nsig_balance.pdf}}
    \captionsetup{font=footnotesize,labelfont=footnotesize}
	\caption{\textbf{Most diseases show microbiome alterations, and consistent disease-associated shifts differ in their extent and direction.} (A) Left: Total sample size for each study included in these analyses. Additional information about each dataset can be found in Table \ref{tab:datasets}. Studies on the y-axis are grouped by disease and ordered by decreasing sample size (top to bottom). Right: Area under the ROC curve (AUC) for genus-level random forest classifiers. X-axis starts at 0.5, the expected value for a classifier which assigns labels randomly, and AUCs less than 0.5 are not shown. ROC curves for all datasets are in Supplementary Figure \ref{fig:roc_curves}. Note that Youngster et al. (2014) \cite{cdi-youngster} had only 4 distinct control patients was excluded from the Random Forest analysis. (B) Left: Number of genera with q $<$ 0.05 (Kruskal-Wallis (KW) test, Benjamini-Hochberg FDR correction) for each dataset. If a study has no significant associations, no point is shown. Right: Direction of the microbiome shift, i.e. the percent of total associated genera which were enriched in diseased patients. In datasets on the leftmost blue line, 100\% of associated (q $<$ 0.05, FDR KW test) genera are health-associated (i.e. depleted in patients relative to controls). In datasets on the rightmost red line, 100\% of associated (q $<$ 0.05, FDR KW test) genera are disease-associated (i.e. enriched in patients relative to controls). Supplementary Figures \ref{fig:overall_heatmap_qvalues} and \ref{fig:overall_heatmap_foldchange} show q-values and effects for each genus in each study.}
	\label{fig:fig1}
	\end{center}
\end{figure}

\newpage
\begin{figure}[h]
	\begin{center}
  	\makebox[\textwidth][c]{\includegraphics[width=0.9\textwidth]{disease_specific_heatmaps.pdf}}
    \captionsetup{font=footnotesize,labelfont=footnotesize}
	\caption{\textbf{Comparing results from multiple studies of the same disease reveals patterns in disease-associated microbiome alterations.} Heatmaps showing log10(q-values) for each disease (Kruskal-Wallis (KW) test, Benjamini-Hochberg FDR correction). Rows include all genera which were significant in at least one dataset within each disease, columns are datasets. Q-values are colored by direction of the effect, where red indicates higher mean abundance in disease patients and blue indicates higher mean abundance in controls. Opacity ranges from q = 0.05 to 1, where q values less than 0.05 are the most opaque and q values close to 1 are gray. White indicates that the genus was not present in that dataset. Within each heatmap, rows are ordered from most disease-associated (top) to most health-associated (bottom) (i.e. by the sum across rows of the log10(q-values), signed according to directionality of the effect). The extent of a disease-associated microbiome shift can be visualized by the number of rows in each disease heatmap; the directionality of a shift can be seen in the ratio of red rows to blue rows within each disease. See Supplementary Figure \ref{fig:supp_dis_specific} for genus (row) labels.}
	\label{fig:fig2}
	\end{center}
\end{figure}

\newpage
\begin{figure}[h]
	\begin{center}
  	\makebox[\textwidth][c]{\includegraphics[width=\textwidth]{core_response.pdf}}
	\caption{\textbf{The majority of disease-associated microbiome alterations overlap with a non-specific microbial response to disease.} (A) Non-specific and disease-associated genera. Genera are in columns, arranged phylogenetically according to a PhyloT tree built from genus-level NCBI IDs (\url{http://phylot.biobyte.de}). Non-specific genera are associated with health (or disease) in at least two different \textit{diseases} (q $<$ 0.05, Kruskal-Wallis (KW) test, Benjamini-Hochberg FDR correction). Disease-specific genera are significant in the same direction in at least two \textit{studies} of the same disease (q $<$ 0.05, FDR KW test). As in Figure \ref{fig:fig2}, blue indicates higher mean abundance in controls and red indicates higher mean abundance in patients. Black bars indicate mixed genera which were associated with health in two diseases and also associated with disease in two diseases. Disease-specific genera are shown for diseases with at least 3 studies. Phyla, left to right: Euryarchaeota (brown), Verrucomicrobia Subdivision 5 (gray), Candidatus Saccharibacteria (gray), Bacteroidetes (blue), Proteobacteria (red), Synergistetes (pink), Actinobacteria (green), Firmicutes (purple), Verrucomicrobia (gray), Lentisphaerae (pink), Fusobacteria (orange). See Supplementary Figure \ref{fig:supp_meta_heatmap} for genus labels. (B) The percent of each study's genus-level associations which overlap with the shared response  (q $<$ 0.05, FDR KW test). Only datasets with at least one significant association are shown. (C) Overall abundance and ubiquity of non-specific genera across all patients in all datasets. Non-specific genera on the x-axis are as defined above.}
	\label{fig:fig3}
	\end{center}
\end{figure}

\bibliographystyle{unsrt}
\bibliography{meta-analysis/refs}
