%% This is an example first chapter.  You should put chapter/appendix that you
%% write into a separate file, and add a line \include{yourfilename} to
%% main.tex, where `yourfilename.tex' is the name of the chapter/appendix file.
%% You can process specific files by typing their names in at the
%% \files=
%% prompt when you run the file main.tex through LaTeX.

\chapter{Introduction}

\section{The human microbiome in health and disease}

The microbes that live in and on our bodies make up the human microbiome, are essential for health, and have been implicated in many diseases.
Almost all human body sites are colonized by microbes, ranging from the gut, the largest human-associated microbial community, to the lungs, which were for many years considered sterile \cite{sender-2016-bhratio,beck-2012-lungmicrobiome}.
These microbial communities perform essential functions for health, including fighting off and preventing infections, regulating host metabolism and interacting with the immune system, and metabolizing xenobiotics or other compounds which are indigestible by the host.
Additionally, perturbations in human microbiomes have been implicated in many diseases, including inflammatory, metabolic, neurological, and respiratory conditions \cite{beck-2012-lungmicrobiome,ibd-papa,ridaura-2013-obfmt,hsiao-2013-autism}.

The potential for microbiome-based therapies to improve human health and address a broad range of diseases has led to a recent expansion of research and clinical studies in this field.
Much of the emerging research has been driven in part by the increasing accessibility of DNA sequencing technology, which can provide a detailed view of the bacteria in these communities without the need for time-consuming and difficult culturing experiments \cite{hmp-2012}.
However, identifying clinically-relevant associations from microbiome studies remains challenging \cite{knights-2011-predictivevalue}.
Microbiome datasets are high-dimensional, with hundreds to thousands of bacterial species measured in usually only tens to hundreds of patient samples.
Microbial communities are also highly variable across people, making it more difficult to identify individual bacterial biomarkers that can consistently distinguish health and disease across many different patients.
Finally, microbiome datasets provide a window into associations but not causal relationships, and researchers must perform follow-up mechanistic studies or clinical trials to confirm the clinical relevance of any identified associations.
In this thesis, I present unique analyses of microbiome data which overcome some of these challenges and which illustrate a variety of approaches for extracting useful clinical insights from mining microbiome data.
Together, the following chapters aim to move findings from the individual microbiome data analyses beyond statistical significance and toward clinical meaningfulness.

\section{Multi-site sampling to identify clinical associations in the aerodigestive microbiome}

In Chapter \ref{chap:aspiration}, I leverage simultaneous sampling within patients to identify clinically-relevant aerodigestive microbiome characteristics that distinguish patients with swallowing dysfunction from those with normal swallow.
The current gold standard to diagnose aspiration resulting from impaired swallow function involves imaging a patient as they ingest different quantities and consistencies of contrast material \cite{cook-1999-dysphagia}, but identifying lung microbial biomarkers could provide a useful non-radioactive alternative to this diagnosis.
Additionally, patients with impaired swallow function are at higher risk for respiratory infections, but the extent to which the lung microbiome is perturbed by aspiration and thus potentially involved in mediating this risk is unknown \cite{cook-1999-dysphagia,thomson-2016-asppneumo}.
Finally, clinical interventions to treat respiratory symptoms in patients with impaired swallow focus on preventing material transfer from the stomach into the lungs, for example via anti-reflux medication or fundoplication surgery \cite{goldin-2006-fundo,lee-2008-fundo}.
However, the clinical utility of these interventions in pediatric populations is not fully established, and the extent to which the gastric vs. mouth microbiome mediates respiratory complications is not known.

In this study, I analyzed a set of aerodigestive microbiome samples collected from over 200 patients at Boston Children's Hospital.
I first show that lung and stomach microbiomes are highly variable across people and driven primarily by person rather than body site, complicating the search for a reliable microbial biomarker of aspiration across patients.
I overcome this challenge by leveraging the fact that we have multiple samples per patient, comparing instead the within-patient \textit{relationships} between microbial communities in the aerodigestive tract of patients with and without aspiration.
Using this approach, I show that aspiration shifts lung microbial communities toward the oropharyngeal but not the stomach microbiome, suggesting that the mouth is likely an important source of lung microbiome changes in these patients.
Thus, approaches for treating aspiration-related respiratory symptoms should target microbial transfer from the mouth into the lungs in addition to focusing on the gastric-lung axis, as most current interventions do.
This study also illustrates the power of multi-site within-patient sampling to overcome variability across people to identify clinically meaningful microbiome-based biomarkers.

\section{Re-analyzing datasets to find consistent patterns of associations between the gut microbiome and disease}

In Chapter \ref{chap:meta-analysis}, I perform a meta-analysis of 28 case-control gut microbiome studies across 10 diseases to synthesize findings across studies and identify generalizable associations.
Although the human gut microbiome has been extensively studied in many diseases, there is little consensus on which disease associations are consistent across patient cohorts.
In other fields like medicine or psychology, consensus is usually achieved through meta-analyses of published literature \cite{glass-1976}.
However, comparing published results across microbiome studies is not straightforward.
The field lacks standard data processing and analysis methods, making many reported results impossible or inappropriate to compare directly between studies.
For example, different studies often use incompatible bioinformatics or statistical methods: they may compare bacteria at different taxonomic levels and use different bioinformatics workflows, and they may identify significant associations through univariate statistical tests or machine learning models, results from which can not be readily compared \cite{edd-singh,crc-baxter,crc-zeller,ob-zupancic}.
Additionally, studies led by clinicians often ask very different questions of the data than studies led by microbial ecologists, and so the reported results are not necessarily representative of the comprehensive information contained in the full dataset \cite{asd-son,ra-scher,nash-wong}.
This issue is especially relevant in this field, where non-invasive sample collection and open-source bioinformatics software suites have made microbiome research accessible to a broad range of scientists and clinicians.

In this chapter, I perform a meta-analysis of gut microbiome studies which overcomes many of these challenges by reprocessing and reanalyzing raw data with standard methods.
Even though specific bacterial associations vary across studies of the same disease, I identify patterns of general microbiome shifts which are consistent  and which suggest different approaches for developing microbiome-based therapeutics.
When looking across multiple diseases, I find a set of bacteria which are non-specifically associated with health and disease, perhaps forming a shared or core response to general health and disease status.
These results highlight the importance of contextualizing results from individual studies within the existing body of work, and also hints at the possibility of developing broadly beneficial targeted probiotic and antibiotic therapies.

\section{Enabling more powerful meta-analyses by developing a batch correction method for microbiome data}

Motivated and enabled by the work presented in Chapter \ref{chap:meta-analysis}, I present a method to correct for batch effects in case-control microbiome studies in Chapter \ref{chap:perc-norm}.
Traditionally, meta-analyses compile published p-values and effect sizes and determine their consistency, significance, and magnitude across studies to glean an overall understanding of true effects \cite{glass-1976}.
However, this is challenging to do in the microbiome research field for reasons described above and in Chapter \ref{chap:meta-analysis}.
A more powerful way to synthesize findings across studies is to combine the raw data and re-perform analyses on this expanded sample size.
However, analyzing raw data combined from multiple studies is not appropriate without first correcting for batch effects.
In microbiome data, batch effects result from biological and technical variation between studies and although statistical methods to correct for these have been developed for other `omics data types, few are applicable for microbiome data \cite{gibbons-2018}.

In this chapter, we describe a method to correct for batch effects in microbiome data.
Briefly, the method converts the abundances of taxa in case samples to percentiles of their respective distributions in the control samples within each study, in essence considering control samples as the null.
Assuming that control patients should represent biologically similar groups, this allows for pooling of percentile-normalized data across studies.
We show that this non-parametric method successfully removes batch effects in case-control studies, enabling more powerful analyses on combined microbiome data and reducing the number of false positives from individual analyses.
My main contributions to this joint work were processing the datasets used to develop and test the method and implementing the method into a popular microbiome software suite, QIIME 2 \cite{qiime2}, expanding its reach and accessibility.

\section{Translating microbiome research through rationally designed fecal microbiota transplant clinical trials}

In Chapter \ref{chap:donor-selection}, I present a framework for rationally selecting donors in fecal microbiota transplant (FMT) clinical trials.
In recurrent \textit{Clostridium difficile} infection (rCDI), FMTs have proven themselves as a remarkably effective clinical treatment, generating excitement about the clinical potential of FMTs in other diseases \cite{quraishi-2017-rcdifmt,bafeta-2017-fmt}.
Clinical trials in non-rCDI diseases can be used to identify promising conditions in which the microbiome may have a causal role to play.
However, although there is little donor-dependent variability in FMT efficacy in rCDI, it is becoming apparent that in many other non-rCDI conditions, donor heterogeneity likely plays a role in patient response \cite{moayyedi-2015,olesen-2018-superstool}.
Thus, as FMT trials expand to new indications, many trials may fail not because the indication was not amenable to improvement by FMT but rather because an ineffective donor was chosen.
As a consequence, excitement and support for FMT research will be dampened, slowing progress toward finding clinical applications of microbiome-based therapies.

In this chapter, we present an approach for selecting donors in FMT trials to increase the likelihood that these trials will succeed.
In our proposed framework, a clinician applies her existing knowledge and hypotheses for how her indication of interest is being mediated by the microbiome to drive the donor selection process.
We present four types of disease models to describe underlying processes mediating microbiome-related diseases: acute dysbiosis, mediation by individual taxa, missing or overactive community function, and complex host-microbiome interactions.
We suggest associated donor selection strategies for each of these disease models, and provide case studies illustrating the process of selecting donors rationally.
Finally, I perform a power simulation which finds that most FMT trials are unlikely to be powered enough to identify the key donor bacteria mediating patient responses to FMT.
Thus, to successfully make discoveries from completed trials, clinicians should perform hypothesis-driven retrospective analyses and plan for these during the clinical trial design process, for example by collecting paired donor and patient and/or longitudinal samples.
The framework that we present encourages clinicians to leverage their clinical experience, existing microbiome research and published datasets, and the increasing availability of screened donor stools to more efficiently translate microbiome research into clinical impact.
It also suggests approaches for improving retrospective analyses of the microbiome data generated during these clinical trials.

\section{Mining the microbiome and metabolome of residential sewage for community-level public health surveillance}

In Chapter \ref{chap:24hr}, I present preliminary work analyzing untargeted metabolomics and microbiome data of residential sewage for human health and activity markers with public health relevance.
Wastewater epidemiology has long been proposed for many applications in population health, but has so far found only limited success in polio surveillance \cite{polio} and illicit drug consumption monitoring \cite{Subedi2014,Ort2014}.
Standard wastewater epidemiology methods sample sewage at the wastewater treatment plant and perform targeted analyses to measure individual metabolites or viruses of interest.
Sampling at the treatment plant can capture information about large urban populations, but does not provide spatially resolved information about individual communities within the city.
This makes current wastewater epidemiology methods inappropriate for many public health applications, since a major goal of public health activities is to reduce disparities between populations by tailoring interventions to specific high-risk groups \cite{Ramaswami2018,Weeramanthri2018,Khoury2016}.
Implementing wastewater epidemiology within communities could be used to evaluate the impact of neighborhood-level public health campaigns or policy changes.
As one example, although it would take many years to measure the impact of a citywide sugar tax on obesity rates, biomarkers of human sugar consumption should quickly decrease upon the implementation of the tax in neighborhoods where it is expected to have a large impact.

In this chapter, we present a platform to implement wastewater epidemiology within cities that mines the metabolome and microbiome of residential sewage to find biomarkers with potential public health relevance.
This project is the culmination of a large collaboration between computational biologists, urban designers, engineers, city workers, and public health officials, and it is presented in its entirety in Chapter \ref{chap:24hr}.
My contribution to this work was identifying and confirming many human biomarkers in the untargeted metabolomics data.
By comparing the masses and fragmentation spectra of our metabolite features with databases and published work, I identified over 20 glucuronide compounds which are direct markers of human excretion and were abundant in our residential sewage samples but rarely detected at downstream sites.
In collaboration with our mass spectrometry specialist, we confirmed multiple urinary and fecal metabolites and showed that these reflected expected human activity patterns over the course of a 24-hour hourly sampling.
Further putative matches between our metabolomics features and a large database of human metabolites \cite{hmdb} suggests that this work could be extended to identify many different types of biomarkers reflecting human health, diet, and lifestyle.

\begin{singlespace}
\bibliography{chap1}
\bibliographystyle{unsrt}
\end{singlespace}
