\section*{Abstract}

Fecal Microbiota Transplantation (FMT) is the transfer of healthy gut bacteria through whole stool into an individual with a diseased microbiome.
Early clinical successes combined with emerging research linking the microbiome to many different diseases are driving enthusiasm for FMT as a clinical treatment for complex conditions.
However, preliminary trials in exploratory indications have yielded mixed results and suggest that heterogeneity in donor stool may play a role in patient response.
Thus, as FMT expands to complex diseases, clinical trials with randomly-selected donors may fail because an ineffective donor was chosen rather than because FMT is not appropriate for the indication.
Here, we describe a conceptual framework to guide rational donor selection to increase the likelihood that FMT clinical trials will succeed.
We argue that the mechanism by which the microbiome is hypothesized to be associated with a given indication should inform how donors are selected for FMT trials.
We describe different disease models which may underlie microbiome-mediated conditions and associated rational donor selection strategies for each, and also provide examples based on previously published FMT trials and ongoing investigations.
Finally, we show that performing discovery-based retrospective research after an FMT clinical trial concludes may require prohibitively large sample sizes, suggesting that clinicians should focus on hypothesis-driven retrospective analyses and plan for these during the clinical trial design process.
In this nascent field, as FMT trials enter new disease indications, a rational approach to donor selection is urgently needed to maximize the insights generated to benefit patients.

\newpage

\section{Introduction}

Fecal Microbiota Transplantation (FMT) is the transfer of gut bacteria through whole stool from a healthy donor to a recipient.
FMT has demonstrated high cure rates in C. difficile infection (CDI) across multiple randomized, placebo-controlled trials \cite{Quraishi2017} and has now entered standard of care for recurrent CDI in European and North American guidelines \cite{McDonald2018,Cammarota2017,Surawicz2013}.
Beyond CDI, FMT is being explored in range of microbiome mediated diseases, and has demonstrated promising results in inflammatory bowel disease \cite{Panchal2018,Gelfand2018,Kootte2017,Osman2018,Costello2017,Paramsothy2017}.

Despite these early successes, the underlying mechanism of FMT across all disease indications, including CDI, remains unclear.
However, it is generally considered that FMT restores gut microbial community perturbations from a dysbiotic to healthy stable state with engraftment of donor strains.
Other donor-dependent features may be important such as the abundance of non-bacterial components or donor clinical features \cite{Ott2017,Zuo2018}.
However, not all FMT donors are alike: gut microbiota compositions vary significantly within healthy populations in ways that could impact the findings from an FMT trial \cite{Yatsunenko2012}.
This critical point of donor microbiome variation is rarely considered in the development of FMT trials \cite{Bafeta2017,Olesen2018}.

Unlike FMT trials in CDI, where selecting donors based on specific clinical or microbiome profiles does not seem to affect clinical response rates, donor selection is likely to be crucial to trial outcomes in diseases with more complex host-microbiome interplay or distinct disease-associated perturbations.
Most notably, in a randomized controlled trial (RCT) of FMT for ulcerative colitis (UC) using 5 donors, 78\% of patients who achieved remission after FMT received stool from a single donor \cite{Moayyedi2015}.
Without this single donor the trial would have returned a negative result.
Given the variation in donor microbiomes and donors' potential impact on clinical efficacy, how should clinicians and investigators select their donors for a clinical trial?

To date, the typical approach for donor selection in FMT trials is to use a single healthy screened donor or to randomly select multiple donors from a set of screened potential donors \cite{Paramsothy2017,Kelly2016,vanNood2013}.
However, in clinical indications where successful donors may be rare, such as UC, clinical trials with randomly-selected donors may fail not because FMT is inappropriate for the indication, but because an ineffective donor was chosen.
An alternative approach is to expose patients to multiple donors in order to mitigate the risk of sub-optimal donor selection.
In a large RCT of FMT in UC, FMT enemas for a single patient were derived from between three and seven donors with patients receiving multiple donors throughout the 8 week course of treatment \cite{Paramsothy2017}.
However, using multiple donors for a single patient may not be feasible or appropriate in many disease indications or clinical trial settings (e.g. single-dose FMT studies).
Continuing the strategy of sub-optimal, random donor selection when it is not warranted risks returning false negative trials, stalling the field and delaying the development of novel therapies for seemingly intractable microbiome-mediated conditions.

Unlike traditional clinical trials which test well-defined small molecules, the therapy under study in FMT trials, the donor microbiome, also varies \cite{Olesen2018}.
Thus, translational FMT research requires a paradigm shift in order to systematically address rational donor selection.
Fortunately, with the emergence of large multi-donor stool banks, expanded access to genome sequencing technologies and publicly available microbiome sequencing datasets, rational donor selection is feasible and presents a unique opportunity to advance the research methods of this nascent field.

In this paper, we present a framework to guide donor selection for FMT trials.
The mechanism by which the microbiome is hypothesized to be associated with a given indication should inform how donors are selected for FMT trials, and we describe different disease models which may underlie microbiome-mediated conditions (Figure \ref{fig:disease-models}).
We describe strategies to rationally select donors for each type of disease model, and provide examples based on previously published FMT trials and ongoing investigations.
Finally, we discuss limitations of performing discovery-based retrospective research after an FMT clinical trial concludes.
To our knowledge, this is the first description of a comprehensive framework for rational donor selection in FMT trials.

\section{Models of microbiome-mediated disease}

FMT trials are pursued when research or clinical experiences suggest that a condition may be causally linked to the microbiome.
Here, we propose four different models which may underlie microbiome-mediated etiologies and their corresponding rational donor selection strategies (Figure \ref{fig:disease-models}).
Ultimately, it is up to each individual clinician-researcher to determine which of these model(s) are relevant in their specific case, based on published cross-sectional studies, mechanistic investigations in model organisms, and their own clinical experience treating patients.
Additionally, logistical considerations will be important factors in making the final donor selection regardless of which strategy is pursued.
Clinicians should ensure that the pool of donors that they are screening have enough material to sustain the required number of FMTs for their entire trial.

Most of the donor selection strategies described below can be modified to incorporate matching between patients and donors.
More specifically, donors can be tailored to individual patients to specifically make up for the unique taxonomic or functional deficiencies in that patient's microbiome.
With the increasing amount of microbiome data available from published FMT trials, we encourage collaborations between clinicians and bioinformaticians to analyze these data in order to generate or perhaps even confirm the validity of potential donor selection strategies before selecting one (Figure \ref{fig:ibd-butyrate}).
Finally, the strategies presented here should also be combined with adaptive clinical trial designs to further increase the probability of having a successful FMT trial \cite{Olesen2017}.

\begin{figure}
    \begin{center}
    \includegraphics[width=\textwidth]{fig1_overview.png}
    \caption{Overview of the different models of microbiome-mediated disease and associated donor selection strategy.}\label{fig:disease-models}
    \end{center}
\end{figure}

\subsection{Acute dysbiosis}

An acutely dysbiotic gut microbial community is broadly dysfunctional and can no longer maintain the health of the host.
For example, in the case of recurrent Clostridium difficile infection, a disturbed microbial community is unable to prevent colonization by the pathogen, leading to recurrent overgrowth of C. diff and clinical symptoms \cite{Britton2014}.
Acute dysbiosis has also been described with the ``Anna Karenina principle'': all healthy microbiomes are alike but dysbiotic communities are all dysbiotic in their own ways \cite{Zaneveld2017}.
In this view of acute dysbiosis, microbial communities respond stochastically to stressors, resulting in dysbiotic communities which are characterized by increased variability rather than deterministic shifts to precise community type(s) \cite{Zaneveld2017}.

In this model, the host just needs to return to a ``healthy'' microbiome and thus choosing any healthy FMT donor should be sufficient to induce clinical improvements.
Because there is no specific disease-associated microbial community and deviation from health is instead the more important factor, simply replenishing the microbiome with a healthy configuration should be sufficient.
Indeed, FMT trials have demonstrated that recurrent C. diff infection can be effectively treated by almost any choice of donor \cite{Osman2016}.
In this case, researchers should consider how they define a ``healthy'' microbiome and how they will ensure engraftment of the transplanted healthy communities.

\subsection{Absence or presence of individual taxa}

\subsubsection{Absence of beneficial taxa}

In other cases, perhaps a disease is being caused or exacerbated by the lack of certain specific microbes, and replenishing these few taxa would be sufficient to restore the host to health.
For example, Hsiao et al showed that a single microbe, \textit{R. obeum}, restricted infection by \textit{V. cholerae} through quorum-sensing-mediated mechanisms \cite{Hsiao2014}.
Surprisingly, non-communicable diseases may also fall into this model: Wilck et al. demonstrated that a single strain of Lactobacillus was sufficient to prevent salt-induced hypertension, and follow-up studies indicate that similar mechanisms may be involved in salt-sensitive high blood pressure in humans as well \cite{Wilck2017}.

In these cases, the donor selection strategy should focus on maximizing the probability of engraftment of the beneficial taxa.
In cases where the unique taxa are not specifically known or are rare members of the human microbiota, many healthy donors should be pooled together to maximize the probability that the transplanted sample contains the necessary taxa.
If the missing microbes are known and well-characterized, on the other hand, researchers can screen their pool of potential donors to find the sample with the highest abundance of these taxa.

\subsubsection{Presence of harmful taxa}

Rather than being characterized by the absence of individual bacteria, perhaps a disease is instead mediated by the presence or overabundance of specific microbes, and removing these bacteria in a targeted fashion could lead to improvements in disease progression.
For example, \textit{Fusobacterium} has been found to be more abundant in colorectal cancer patients, specifically enriched in the tumors themselves \cite{Kostic2013}.
Multiple groups have identified mechanistic associations between \textit{Fusobacterium}, inflammatory transcriptional signatures, and tumor growth in mouse and human models of colorectal cancer, pointing to a causal role for \textit{Fusobacterium} in colorectal cancer progression \cite{Kostic2013,Rubinstein2013}.
Recent work has found that treating tumors with antibiotics slows tumor progression, further confirming these causal associations and pointing toward potential microbiome-based therapeutic interventions \cite{Bullman2017}.

Removing and replacing these bacteria should be the goal of FMT in cases where this disease model applies.
This can be achieved by first removing the harmful bacteria in a targeted way (e.g. via antibiotic treatment) with follow-up FMT to re-establish a healthy community that prevents their re-colonization.
In all cases, donors should be screened to exclude any samples which contain the harmful bacteria.
Donor samples can then be selected based on the abundance of bacteria which are known to out-compete the harmful taxa.
Competitors can be identified by searching the microbiology literature to identify bacteria which live in the same niche or which have been experimentally shown to directly out-compete the undesirable taxa, or they can perform these competition assays themselves.
If resources to perform competition assays are not available and the literature is sparse, researchers can also mine existing microbiome data to find bacteria which consistently anti-correlate with the harmful taxa, and choose donor samples with a high abundance of these putative competitors.

\subsubsection{Patient matching}

Taxa-based donor selection strategies are particularly amenable to patient-matching, when both patient and donor microbiome data are available prior to the start of a trial.
For example, if one patient is completely missing some of the beneficial taxa but not others, then these taxa can be weighted more heavily in the donor selection process.
The phylogenetic relationships between donor and recipient taxa could also be incorporated into donor selection: if a patient already has many bacteria which are closely phylogenetically related to known competitors of some of the harmful bacteria, then competitors of the other harmful bacteria can be upweighted in the donor selection process.
Similarly, if patients already have taxa which are already filling certain niches important for health, the taxa which fill those same niches can be downweighted in donor selection.

\subsubsection{Case study: Inflammatory Bowel Disease}

An example where the ``missing taxa'' model may be applicable is in inflammatory bowel disease (IBD).
Butyrate has long been associated with inflammatory bowel disease \cite{Scheppach1992}, and recent case-control and longitudinal studies point to a consistent lack of butyrate-producing bacteria in patients with IBD \cite{Duvallet2017,Schirmer2018}.
Furthermore, preliminary FMT trials in IBD have been marked by significant donor variability and suggest that donor microbiome characteristics may be associated with FMT response \cite{Moayyedi2015,Kump2018}.
These results indicate that IBD may benefit from rational donor selection approach, and that donors with high abundances of butyrate-producing organisms may yield higher FMT response rates than randomly selected donors.

Given the availability of microbiome data from completed FMT studies, we tested this hypothesis that IBD trials would benefit from a rational donor selection strategy based on the ``absence of beneficial taxa'' disease model.
We re-analyzed microbiome data from three completed IBD FMT trials which provided publicly available sequencing data for patient and donor samples \cite{Kump2018,Goyal2018,Jacob2017}.
We selected butyrate-producers based on their genus-level taxonomy, using the genera identified in Vital et al. 2017 (see Methods, \cite{Vital2017}).
Donors in the three studies exhibited a range of total abundances of butyrate-producing bacteria (Figure \ref{fig:ibd-butyrate}A).
Surprisingly, however, the abundance of butyrate producers in the donor stool was not associated with recipient patients' clinical responses (Figure \ref{fig:ibd-butyrate}B).
We also found no association with response when matching donor abundances with their respective patient's original abundance of butyrate producers (Supplementary Figure \ref{fig:delta-butyrate}).
These results show that selecting donors based on the abundance of butyrate producers may not yield improved clinical trial outcomes in IBD, and illustrates the process by which clinicians could approach and validate a rational donor selection strategy based on individual taxa.
More complex methods to identify butyrate producers (e.g. using phylogenetic-aware methods and/or metagenomics data) could be used in the next iteration to develop a donor selection strategy, if these data are available to clinicians.
Another approach, discussed below, is to select donors based on functional community assays and direct measurement of butyrate production rather than microbial taxonomies alone.

\begin{figure}
    \begin{center}
    \includegraphics[width=\textwidth]{fig2_ibd_butyrate.png}
    \caption{Case study in IBD: select donors based on abundance of butyrate producers? (A) abundance of butyrate producers in each study's donor samples. (B) abundance of butyrate producers in donor samples, stratified by respective patient's response.}\label{fig:ibd-butyrate}
    \end{center}
\end{figure}

\subsection{Community-level functionality}

Some microbiome-associated diseases may not be addressable by replenishing the patient with a generically healthy community or by targeting individual taxa, and may instead be mediated by the microbiome through a community-level function.
Here, there may not be a consistent disease-associated microbiome across patients in terms of taxonomic composition, but patients may be characterized by having microbiomes which are similarly missing or enriched in some core functionality.
This model may also apply to conditions where there are consistent disease- or health-associated taxa, but in which their collective functioning is the more important mediator of disease.
The IBD case study described above may reflect this situation: although depletion of butyrate producers is strongly associated with IBD throughout the literature, a successful donor selection strategy may need to consider butyrate production directly rather than through the proxy of taxonomy \cite{Duvallet2017,Schirmer2018}.

\subsubsection{Missing community-level function}

In the case where a community-level function is missing from patients' microbiomes, the goal of FMT should be to replace the deficient community with a beneficially functional microbiome.
Here, it is important that a single donor with an intact microbial community is used, rather than a mixture of donors which may not yield the desirable community composition at steady-state after engraftment.
To choose a donor, molecules which can serve as proxies for the metabolic output of the microbial community can be measured directly in donor stool, and donors can be selected based on the abundance of these molecules.

Like IBD, hepatic encephalopathy (HE) is an example where community functionality is likely more relevant to FMT outcome than specific taxa.
A previous trial in HE \cite{Bajaj2017} rationally selected their single donor by maximizing the abundance of \textit{Lachnospiraceae} and \textit{Ruminococcaceae}, taxa which were identified based on cross-sectional microbiome data.
The clinical trial was a success, but it remains unclear from this trial whether the donor's strains engrafted in the patients post-FMT and whether this played any role in the successful FMT responses.
The exact mechanisms of action of these strains remains unknown, though both bacterial families are known short chain fatty acid producers (in particular butyrate) \cite{Vital2017}.
More recent studies have more directly implicated the production of short chain fatty acids and secondary bile acids as being important in liver cirrhosis and subsequent complications such as HE, suggesting that community-level functioning may be a more important driver of FMT response.
Thus, HE may be a case in which function-based donor selection can be employed.

To illustrate this process, we analyzed stool metabolomics data from 83 OpenBiome donors and used this data to rank them based on their estimated production of short chain fatty acids and secondary bile acids (Figure \ref{fig:bn10-liver}).
As in the IBD case study, we found that donors exhibited a range of values for our metabolites of interest (Figure \ref{fig:bn10-liver}A and C).
We ranked donors based on their amounts of the three measured short-chain fatty acids (butyrate, isovalerate, and propionate) and on their bile acid conversion rates, approximated as the ratio between the total amounts of primary and secondary bile acids (Figure \ref{fig:bn10-liver}B and D).
With this process, we were able to identify four donors who were in the top 25\% of all donors for both metrics (Figure \ref{fig:bn10-liver}E).
In a real FMT trial, a clinician would then work with their stool bank to ensure that these donors were still active and/or had enough material to be used in the full trial.

While measuring metabolites in stool as a proxy for community production will likely be an improvement over taxonomy-based approaches in most cases, these measurements are also complicated by potential host effects.
For example, different hosts may absorb these molecules at different rates, and so measuring them in stool may not be an accurate reflection of each donor community's productive potential.
Additionally, community function may depend on non-biologically relevant factors like the donor's diet and time that they provided their sample. As an example, bile acid production spikes after meals \cite{Hofmann1989}, so the amount of bile acids measured in a given stool sample may reflect the amount of time since the donor last ate rather than their actual microbial community's functional production of these molecules.
If clinicians have access to sufficient resources, a better way to screen donors may be to perform ex-vivo assays, in which each donor sample is homogenized and provided with the substrates (e.g. fiber) needed to produce the desirable output (e.g. short-chain fatty acids like butyrate).
In this way, the donor community function can be measured directly \cite{Wang1993,Chen2017}.

\begin{figure}
    \begin{center}
    \includegraphics[width=\textwidth]{fig3_cirrhosis_metabolomics.png}
    \caption{Case study in liver cirrhosis: selecting donors based on community function by mining stool metabolomics data. (A) Distribution of SCFAs in all donor stools. (B) Abundance of each SCFA per donor, ranked by average SCFA abundance. (C) Distribution of bile acids in all donors. Primary bile acids are in the left column, secondary bile acids are in the right column. Bile acids are colored according to pathways. (D) Bile acid conversion ratios in each donor, ranked by total secondary to primary bile acids. (E) The five donors in the top 25\% for both of these metrics, for example, could be used in a rationally-designed liver cirrhosis FMT trial.}\label{fig:bn10-liver}
    \end{center}
\end{figure}

\subsubsection{Overactive function}

A disease may also be mediated by an overactive microbiome doing something harmful to the host.
For example, many studies have shown a causal association between TMAO produced by the microbiota and atherosclerosis \cite{Koeth2013,Wang2015}.
Here, the goal of FMT should also be to replace the patient's microbiome with a beneficially functional community, but the donor selection strategy may attempt to identify communities in which the harmful function is completely absent or which produces an inhibitor of the harmful microbe-derived molecule \cite{Wang2015}.

\subsection{Microbiome-associated host phenotypes}

Diseases with more complex etiologies may not have a direct taxonomic or functional association with the microbiome but instead be related through some intermediate host phenotype which needs to be improved or corrected.
For example, severe acute malnutrition has been associated with a gut microbiota which is not fully mature, with mouse experiments suggesting that this association may be causal \cite{Blanton2016,Subramanian2014}.
Other studies have shown a relationship between gut microbiome, immune development, and development of autoimmune conditions later in life \cite{Stokholm2018,Cox2014,Kostic2015}.
These relationships may have mechanistic explanations which are not directly measurable from donor or patient stool (e.g. immunogenicity of bacteria, ability of bacteria to eat the host's mucus, etc) but which can nevertheless be inferred from existing data and used to select potential donors.

For these more complex cases, models can be trained from existing datasets to learn the community signatures linked to the disease-associated phenotype. In some cases, it may be possible to develop computational models which directly predict the phenotype of interest.
For example, Stein et al. developed a model to predict the induction of regulatory T-cells by microbial communities \cite{Stein2018}.
In other cases with few known mechanistic models, machine learning algorithms can be trained on multiple cross-sectional datasets to identify complex signatures that reproducibly distinguish patients from healthy controls.
These models can then be applied to score potential donors, and the donor with the ``most healthy'' score may be chosen for a trial.

\section{Little understanding of underlying disease model}

In some conditions, there may not be enough understanding of the underlying microbiome-based etiology to inform donor selection in an FMT trial. It may also be the case that there are no existing datasets on which to train models, existing datasets are not sufficiently powered to distinguish the different potential underlying models, or logistical considerations constrain a clinician's ability to select specific donors for their trial. In these cases, we recommend selecting different healthy donors, employing an adaptive clinical trial design in which donors are cycled after they have clinical failures (as described previously in Olesen, Gurry, and Alm 2017), and performing retrospective analyses to answer targeted hypotheses which were developed during the clinical trial design process.

\subsection{Cycling healthy donors in adaptive trials}

As donors change through the course of an adaptive trial, clinicians may elect to select their donors randomly or to more rationally cycle through donors (Olesen, Gurry, Alm 2017). ``Differently healthy'' donors may be selected, perhaps representing different underlying disease-associated models described above. Donors may also be selected to span the range of ``healthy'' microbiomes in a given population. For example, clinicians may pick a ``median'' health donor (i.e. one who is about as similar to all reference microbiomes), define a ``healthy plane'' and pick donors based on their distance to this plane (as in Halfvarson et al 2017), or simply based on the presence or abundance of certain consistently ``core'' health-associated bacteria (Shade and Handelsman 2012; Duvallet et al. 2017). In a similar vein, ``healthy'' donors can also be chosen based on their distance from disease-associated microbiomes, as opposed or in addition to similarity to health. Published case-control datasets can be used to identify donors with communities which are farthest away from the median or average diseased patient. These datasets can also be mined to identify taxa which are consistently disease-associated, and which should be minimized or perhaps even absent in the potential donor. Pairing rational donor selection with adaptive trial designs may eventually yield insight into the underlying model mediating the disease of interest if certain types of ``healthy'' donors consistently perform better at treating patients than others.

\subsection{Discovery-based retrospective analyses}

In these exploratory FMT clinical trials, discovering microbiome characteristics which are differentially associated with FMT response may be a valuable secondary endpoint \cite{Olesen2018}, identifying characteristics of good donors and informing donor selection strategy for future trials.
Furthermore, companies attempting to develop microbiome-based therapeutics may use FMT trials to discover the key bacteria which mediate FMT response in order to include these in their microbial cocktail product.
However, exploratory FMT trials tend to enroll few patients, limiting the potential power of retrospective analyses to find associations between the microbiome and FMT response.

We performed a simulation to determine the likelihood of a retrospective analysis to identify donor-derived bacteria associated with different patient responses to FMT.
We performed this simulation for multiple FMT trial set-ups and outcomes (i.e. number of FMT responders and non-responders).
We used existing microbiome datasets to model different effect sizes, where we use ``effect size'' to mean the number of bacteria which are differentially abundant in donor samples given to patients who did and did not respond to FMT.
We used case-control datasets to model the microbiome data and various effect sizes, with a large effect represented by an infectious diarrhea dataset \cite{Schubert2014}, a medium effect represented by colorectal cancer \cite{Baxter2016}, and a weak effect represented by obesity \cite{Goodrich2014}.
For each of these datasets, we identified the top ten most differentially abundant bacteria in the overall population as the key mediating bacteria (see Methods).
Next, we simulated different trials, varying the numbers of patients in the FMT arm and the FMT response rates (i.e. proportion of patients which were FMT responders, represented by sampling from the ``case'' patients, vs. non-responders, represented by sampling from the ``control'' patients, representing non-responders).
We subsampled patients according to these parameter settings, identified differentially abundant genera, and compared these to the top ten genera identified from the entire datasets (Figure \ref{fig:power-sim}).

In cases where the microbial signature for FMT response is expected to be large (i.e. the difference in donor stools given to FMT responders vs. non-responders is as large as the effect of diarrhea effect on the microbiome), we found that small FMT trials would recover most of the top hits in most cases.
The power to detect associations decreased as FMT response rates became less balanced (i.e. response rates different from 50\%), and in these cases trials would need to include up to 50 patients in the FMT arm to recover the key mediating taxa.
For both medium and small effect sizes, however, prohibitively large FMT arms would be needed to recover most key mediating taxa.
We found that when the microbial signature for FMT is equivalent to the effect of diseases like colorectal cancer on the microbiome, at least 100 patients are needed in the FMT arm to recover at least half of the most truly differentially abundant genera for most FMT trials.

These results suggest that successful secondary analyses of microbiome data from FMT trials will require either very large FMT arms, investigating more targeted hypotheses, or additional sample collections.
For example, clinicians may consider pairing donor and patient samples or collecting longitudinal patient samples to increase power to make discoveries.
They may also consider testing specific hypotheses developed before the trial, such as comparing the total abundance of butyrate producers between FMT responders and non-responders, or performing functional assays to measure specific metabolites thought to be associated with FMT response.
On the other hand, researchers wishing to identify the key taxa to include in an FMT drug may consider pursuing clinical trials in which identifying these taxa is the primary endpoint, and power them accordingly.


\begin{figure}
    \begin{center}
    \includegraphics[width=\textwidth]{fig4_power_simulation.png}
    \caption{Power simulation results, showing how many of the 10 most ``truly'' differentially abundant genera would be recovered as significant under different FMT study designs. Each panel represents a different FMT response rate (i.e. percent of patients in the responder vs. non-responder group). The effect size (i.e. number of genera which are differentially abundant in FMT responders vs. non-responders) was simulated by using three different case-control microbiome datasets. A large effect size is modeled by the effect of diarrhea on the microbiome, medium by colorectal cancer, and small by obesity. The top 10 "true" differentially abundant genera were identified by calculating their signal-to-noise ratios in the full original dataset (i.e. mean difference divided by the standard deviation).}\label{fig:power-sim}
    \end{center}
\end{figure}

\section{Discussion}

The framework presented here encourages clinicians to leverage their clinical experience, existing microbiome research and published datasets, and the increasing availability of screened donor stools to more efficiently translate microbiome-based interventions into clinical impact.
Clinicians can apply their existing knowledge and a priori hypotheses to determine which microbiome-mediated disease model may underlie their indication of interest, and then select donors accordingly.
By rationally choosing donors during the FMT trial design, clinicians will increase the likelihood of successful FMT trials in diseases in which donor heterogeneity affects patient response.
Our power simulation analysis also suggests that retrospective analyses of the microbiome data generated should be developed during trial design, with targeted hypotheses of interest and sample collection plans tailored accordingly.
Otherwise, untargeted discovery-based analyses are unlikely to yield key information from most FMT trials.
Paired with adaptive clinical trial designs, FMT trials with rationally-selected donors will become an important tool in advancing translational microbiome research and clinical treatment to improve and save patient lives.

As clinical trial design methodologies for FMT trials become more developed, many additional questions will need to be addressed.
Many of these key questions relate to the nuances involved in choosing healthy donors: what defines a ``healthy'' donor, and what should define one?
These questions are critical because regardless of the underlying model, in all cases a healthy donor must be identified.
However, as our understanding of the microbiome in societies around the world continues to increase, consensus on the exact structure of a ``healthy'' microbiome decreases.
Should donors be selected to reflect the patient populations, or simply be devoid of pathogens and source from as ``healthy'' of donors as possible?
We know that Europeans and North Americans tend to have less \textit{Prevotella} than healthy Africans from across the continent in both urban and rural communities \cite{Yatsunenko2012,Ou2013,DeFilippo2010}.
Thus, should a clinician carrying out an FMT trial in an African setting use local African cohort as their donor population in order to better match the expected ``healthy'' state of their participants?
In some cases, using a local population may be incompatible with established donor screening criteria because of geographical differences in baseline microbial community composition, prior likelihood of pathogen colonization, or differences in diet and lifestyle habits leading to different dominating strains.
Should clinicians consider relaxing donor screening and exclusion criteria in cases where donors are sourced from countries where commensal colonization by potential pathogens is common population-wide?
More research to understand the functional differences and clinical implications of different ``healthy'' communities across global populations and specifically in the context of FMT trials will be required before these and many related questions can be answered \cite{Bello2018,Rabesandratana2018}.

On the patient side, comorbidities, lifestyle, and dynamic disease manifestations may present challenges and opportunities to improve donor selection processes.
How should comorbidities be incorporated into donor selection?
Patients with multiple disease processes may be dominated by one microbiome-mediated disease model or may exhibit a combination of models, perhaps affecting which donors would be optimal for their specific case.
For example, a person with one community-level function process and one acute dysbiosis process may respond well to total community replacement alone, or may require a combination of total community replacement along with enrichment for community function.
Additionally, will diseases that change manifestations over time benefit from employing different donor selection strategies over the course of a longitudinal FMT trial?
Although there have been no serious adverse events related to FMT material in either clinical practice for rCDI or in clinical trials across adults or pediatrics, could some donors further reduce the probability of adverse events in at-risk patients?
Finally, how should other sources of heterogeneity like lifestyle, diet, and medication usage be incorporated into rational donor selection?
In cases where FMT is combined with other microbiome-targeted interventions like prebiotics or dietary changes, could some donors have synergistic effects with these paired interventions and lead to greater clinical success?

To ensure that FMT reaches its full potential to improve and save patient lives, clinicians should think critically about how their FMT trials can be designed for maximal impact.
By applying new approaches like rational donor selection and adaptive trial designs, the number of trials which fail even though they could have succeeded will drastically decrease.
Furthermore, by developing targeted hypotheses, post-trial analysis plans, and associated sample collection schema alongside the core FMT trial design itself, the number of basic scientific discoveries that are made from each trial will significantly increase.
As FMT expands far beyond rCDI and microbiome-based therapeutics are developed to target a range of disease, novel methods and approaches tailored to the unique challenges and opportunities presented by FMT will be critical to ensuring the advancement of translational microbiome science and beneficial impact on patient lives.

\section{Methods}

\subsection{Microbiome data processing}

Raw fastq data files were downloaded from the European Nucleotide Archive using the following accession numbers: Jacob et al 2017, PRJNA388210; Goyal et al. 2018, PRJNA380944; and Kump et al. 2018, PRJEB11841.
All data was processed using QIIME 2 (v. 2018.6.0, \cite{qiime2}).
Briefly, data was imported into QIIME 2 as paired-end (Kump et al. 2018; Jacob et al. 2017) or single-end (Goyal et al. 2018) data, filtered based on sequence quality with \texttt{quality-filter q-score}, and denoised with deblur using \texttt{deblur denoise-16S} \cite{deblur}.
Representative sequences were assigned taxonomy using \texttt{feature-classifier classify-sklearn} with the GreenGenes-trained Naive Bayes classifier provided by QIIME 2 (gg-13-8-99-nb-classifier.qza) \cite{feature-classifier}.
All data was exported to tab-delimited format and analyzed in Python 2.7.6.

\subsection{Quantifying abundance of butyrate producers}

We identified butyrate producers at the genus-level based on the analysis performed in Vital et al. 2017 \cite{Vital2017}.
These taxa were detected in >70\% of individuals in Vital et al. 2017, are known butyrate producers (with a majority of the analyzed representative genomes containing known butyrate production pathways), and accounted for the majority of the total butyrate pathway abundances in human metagenomics data.
We removed \textit{E. ventriosum}, \textit{E. hallii}, and \textit{E. rectale} from our analyses as these species-level taxa do not comprise one genus with conserved butyrate production.

\subsection{Stool metabolomics}

Metabolomics data was generated as described in Poyet, Groussin, Gibbons et al. (in preparation) and downloaded after personal communication with the authors.
For donors with multiple samples, we considered the mean metabolite abundances across all sampled time points.
We identified three short chain fatty acids in the data (propionate, butyrate, and isovalerate) and the major primary (cholate and chenodeoxycholate) and secondary (deoxycholate and lithocholate) bile acids.
Lithocholate was available for both C-18 negative and HILIC negative modes; we considered only the C-18 negative data to match the other bile acids.
Bile acid conversion rates were calculated as in Kakiyama et al. 2013 \cite{Kakiyama2013}.
Donors were ranked based on their average SCFA abundances and based on the total bile acid conversion ratio ( (lithocholate + deoxycholate) / (chenodeoxycholate + cholate) ).

\subsection{Power simulation}

We performed a simulation to determine the power of FMT trials to retrospectively find associations between donor bacterial abundances and FMT response.
We used case-control gut microbiome datasets from MicrobiomeHD \cite{Duvallet2017} to model different effect sizes for FMT response.
Here, we use ``effect size'' to mean the number of genera which are differentially abundant between patients who respond to FMT vs. patients who do not.
Per the results in MicrobiomeHD, we used infectious diarrhea to model a large effect \cite{Schubert2014}, colorectal cancer to model a medium effect \cite{Baxter2016}, and obesity to model a small effect \cite{Goodrich2014}.
We collapsed OTUs to genus-level as in \cite{Duvallet2017} and ranked genera according to their signal-to-noise ratio in each entire dataset, where the signal-to-noise was calculated as the difference in mean log abundance in cases and controls divided by the standard deviation of the log abundances across all samples.
We considered the 10 genera with the largest absolute signal-to-noise ratios as our ``top hits'' in the main text.

We modeled different FMT clinical trial designs and outcomes by varying the number of total patients in the trial and the percent of FMT responders (i.e. the number of patients we selected from the original ``case'' group relative to the original ``control'' patients, to model FMT responders and non-responders).
For each of these designs, we subsampled the correct number of case samples to represent FMT responders and control samples to represent non-responders from the original datasets.
We identified significantly differentially abundant genera with the \texttt{kruskalwallis} function from \texttt{scipy.stats.mstats} (scipy v. 1.1.0) as genera with q $<$ 0.05 after multiple hypothesis testing correction with the \texttt{multipletests} function (\texttt{method=`fdr\_bh'}) from the \texttt{statsmodels.sandbox.stats.multicomp} package (statsmodels v. 0.9.0).
We then counted how many of the top genera identified through the signal-to-noise ranking were identified as significantly different as a proxy for the power to detect effects.

\subsection{Code and data availability}

Code to reproduce all of these analyses and figures can be found at \url{https://github.com/cduvallet/donor-selection/}.
Data were downloaded from original sources as described above.

\newpage
\section{Supplementary Figure}

\FloatBarrier
\begin{figure}[h]
    \begin{center}
    \includegraphics[width=\textwidth]{suppfig_delta_butyrate_vs_response.png}
    \caption{Difference between abundance of butyrate producers in donor sample and respective patient sample, stratified by patient response.}\label{fig:delta-butyrate}
    \end{center}
\end{figure}

\begin{singlespace}
\bibliographystyle{unsrtnat}
\bibliography{donor-selection/donor-selection-refs.bib}
\end{singlespace}
